Cargo.toml

```diff
@@ -26,6 +26,7 @@ edition = "2024"
 rpath = true
 
 [profile.release]
+lto = true
```

Comment:
This enables LTO for all binaries in the workspace. I'd recommend doing the [following](https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles):

1. Create a separate profile:
```toml
[profile.release-lto]
inherits = "release"
lto = true
````
2. use `release-lto` profile in `edr_napi/package.json`: `cargo build --profile release-lto` instead of `--release`

I looked at overriding lto for a single package (from the workspace Cargo.toml) but [it's not supported](https://doc.rust-lang.org/cargo/reference/profiles.html#overrides):

> Overrides cannot specify the `panic`, `lto`, or `rpath` settings.